VCFv4.3.tex

```diff
@@ -324,9 +324,9 @@ \subsubsection{Fixed fields}
   `0' is reserved and must not be used as a filter String.
   If filters have not been applied, then this field must be set to the MISSING value.
   (String, no white-space or semi-colons permitted, duplicate values not allowed.)
-  \item INFO --- additional information: (String, no semi-colons or equals-signs permitted; commas are permitted only as delimiters for lists of values; characters with special meaning can be encoded using the percent encoding, see Section~\ref{character-encoding}; space characters are allowed)
+  \item INFO --- additional information: (String, no semi-colons or equals-signs permitted; commas are permitted only as delimiters for lists of values; characters with special meaning can be encoded using the percent encoding, see Section~\ref{character-encoding}; space characters are allowed). If there is no additional information, then the MISSING value must be used.
```

Comment:
This change looks good to me, but could you put the new sentence on a new line?  That's the newer formatting style we're using.  so that it makes it easier to see diffs in the tex.

Reply:
Done!